Kickstarter-iOS/Views/Cells/DiscoveryProjectCardCell.swift

```diff
@@ -0,0 +1,425 @@
+      |> ksr_addSubviewToParent()
+
+    _ = ([
+      self.projectNameLabel,
+      self.projectBlurbLabel,
+      self.projectInfoStackView
+    ], self.projectDetailsStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.goalMetIconImageView, self.percentFundedLabel], self.goalPercentFundedStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.backersCountIconImageView, self.backersCountLabel], self.backersCountStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.goalPercentFundedStackView, self.backersCountStackView], self.projectInfoStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+  }
+
+  private func setupConstraints() {
+    _ = [
+      self.projectImageView,
+      self.projectDetailsStackView,
+      self.goalMetIconImageView,
+      self.backersCountIconImageView,
+      self.saveButton
+    ]
+      ||> \.translatesAutoresizingMaskIntoConstraints .~ false
+
+    let aspectRatio = CGFloat(9.0 / 16.0)
+
+    let imageHeightConstraint = self.projectImageView.heightAnchor.constraint(
+      equalTo: self.projectImageView.widthAnchor,
+      multiplier: aspectRatio
+    ) |> \.priority .~ .defaultHigh
+
+    NSLayoutConstraint.activate([
+      self.projectImageView.topAnchor.constraint(equalTo: self.cardContainerView.topAnchor),
+      self.projectImageView.leftAnchor.constraint(equalTo: self.cardContainerView.leftAnchor),
+      self.projectImageView.rightAnchor.constraint(equalTo: self.cardContainerView.rightAnchor),
+      self.projectImageView.widthAnchor.constraint(equalTo: self.cardContainerView.widthAnchor),
+      imageHeightConstraint,
+      self.saveButton.heightAnchor.constraint(equalToConstant: Styles.minTouchSize.height),
+      self.saveButton.widthAnchor.constraint(equalToConstant: Styles.minTouchSize.width),
+      self.saveButton.topAnchor.constraint(equalTo: self.cardContainerView.topAnchor),
+      self.saveButton.rightAnchor.constraint(equalTo: self.cardContainerView.rightAnchor),
+      self.projectDetailsStackView.topAnchor.constraint(equalTo: self.projectImageView.bottomAnchor),
+      self.projectDetailsStackView.leftAnchor.constraint(equalTo: self.cardContainerView.leftAnchor),
+      self.projectDetailsStackView.rightAnchor.constraint(equalTo: self.cardContainerView.rightAnchor),
+      self.projectDetailsStackView.bottomAnchor.constraint(equalTo: self.cardContainerView.bottomAnchor),
+      self.goalMetIconImageView.widthAnchor.constraint(equalToConstant: IconImageSize.width),
+      self.goalMetIconImageView.heightAnchor.constraint(equalToConstant: IconImageSize.height),
+      self.backersCountIconImageView.widthAnchor.constraint(equalToConstant: IconImageSize.width),
+      self.backersCountIconImageView.heightAnchor.constraint(equalToConstant: IconImageSize.height)
+    ])
+  }
+
+  private func attributedString(bolding boldedString: String, in fullString: String) -> NSAttributedString {
```

Comment:
This might also be achievable with the extension in `String+Attributed.swift`.

Reply:
Ah unfortunately we're not just bolding the font but also using a slightly different one as well for the "larger" bolded string.